docs/utilities/recipes.md

```diff
@@ -113,15 +113,20 @@ The following example shows how an ingredient list looks like within JSON.
     ],
 ```
 
+### Smelting
+To define a recipe for the furnace, you have to define it with the type `minecraft:smelting`. This type of recipe takes in an ingredient and returns a result item. Optionally a `experience` modifier may be applied as a value between 0 and 1. The time in ticks it takes to smelt the item can be defined using `cookingtime` and will default to 200.
```

Comment:
Is this 1.13 information? If so, leave it for when that update is actually usable with Forge...

Reply:
After looking at it again.. Yeah this how Mojang did it in 1.13. Just missed the fact that it isn't available in 1.12 yet. ^^